### PR TITLE
added jackpot/prize api documentation

### PR DESCRIPTION
**Pull request by FernandoArevaloE**

### CODE REVIEW

File: app/src/docs/init.json [hunk 1]
```diff
@@ -694,13 +736,24 @@
             }
           }
         ],
+        "requestBody": {
+          "description": "",
```
Comment (andymotagl) [inline app/src/docs/init.json:740]: Can this have a value?

File: app/src/docs/init.json [hunk 2]
```diff
@@ -694,13 +736,24 @@
...
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/RedeemedPrizeRqst"
```
Comment (andymotagl) [inline app/src/docs/init.json:744]: This can be RedeemedPrizeRequest. It's documentation :stuck_out_tongue:

File: app/src/docs/init.json [hunk 3]
```diff
@@ -1476,6 +1729,104 @@
           }
         }
       },
+      "Jackpot": {
+        "type": "object",
+        "properties": {
+          "jackpot_id": {
```
Comment (andymotagl) [inline app/src/docs/init.json:1735]: Let's use camelCase here

File: app/src/docs/init.json [hunk 4]
```diff
@@ -1476,6 +1729,104 @@
...
+          "company_id": {
+            "type": "string"
+          },
+          "createdAt": {
+            "type": "string",
+            "format": "date-time"
+          }
+        }
+      },
+      "NewJackpot": {
```
Comment (andymotagl) [inline app/src/docs/init.json:1750]: You can remove this and make the required fields on Jackpot definition. See Actor for reference

File: app/src/docs/init.json [hunk 5]
```diff
@@ -1476,6 +1729,104 @@
...
+          "active": {
+            "type": "boolean"
+          },
+          "createdAt": {
+            "type": "string",
+            "format": "date-time"
+          }
+        }
+      },
+      "NewPrize": {
```
Comment (andymotagl) [inline app/src/docs/init.json:1800]: Same as above

File: app/src/docs/init.json [hunk 6]
```diff
@@ -1476,6 +1729,104 @@
...
+          "createdAt": {
+            "type": "string",
+            "format": "date-time"
+          }
+        }
+      },
+      "NewJackpot": {
+        "required": [
+          "name",
+          "company"
```
Comment (andymotagl) [inline app/src/docs/init.json:1753]: Should not be companyId?

**Review by andymotagl — APPROVED** [review]
Beside that, lgtm!

File: app/src/docs/init.json (hunk 6 above)
Comment (FernandoArevaloE) [reply app/src/docs/init.json:1753]: well, I'm using the value that we have in the router. Should we change the router to `req.body.companyId`?

File: app/src/docs/init.json (hunk 2 above)
Comment (FernandoArevaloE) [reply app/src/docs/init.json:744]: done

File: app/src/docs/init.json (hunk 3 above)
Comment (FernandoArevaloE) [reply app/src/docs/init.json:1735]: done

File: app/src/docs/init.json (hunk 6 above)
Comment (andymotagl) [reply app/src/docs/init.json:1753]: I remember we had this conversation to denote only the component without `Id` but the documentation can bring another approach, gonna open the discussion with the team.

Comment (FernandoArevaloE) [reply app/src/docs/init.json:1753]: ok. For now let's change to include Id then we can update if required

**Pull request merged**